### PR TITLE
Add endpoints for classes

### PR DESCRIPTION
**Pull request by theodorusandi**

### CODE REVIEW

**Review by jdmedlock — APPROVED** [review]
This looks good to me. I especially like the fact you've used `Object.freeze`, I think this is overlooked by many Devs (including myself).

File: middlewares/auth.middleware.js [hunk 1]
```diff
@@ -12,9 +13,10 @@ const isAdmin = (req, res, next) => {
   if (req.isAuthenticated() && req.user.isAdmin) {
     next();
   } else {
+    console.log(req.user);
     res
       .status(401)
-      .json({ msg: "You are not authorized to view this resource." });
+      .json({ msg: "You are not admin" });
```
Comment (RDiMag) [inline middlewares/auth.middleware.js]: Can we update this wording to this? 
"You do not have admin permissions."

**Review by RDiMag — APPROVED** [review]

File: server.js [hunk 2]
```diff
@@ -16,16 +16,16 @@ const app = express();
...
-//   const corsOptions = {
-//     origin: "https://100devstracker.netlify.app",
-//     optionsSuccessStatus: 200,
-//   };
-//   app.use(cors(corsOptions));
-// } else {
-//   app.use(cors());
-// }
-app.use(cors());
+if (process.env.NODE_ENV === "production") {
```
Comment (jpatrici) [inline server.js]: does it count as production if we call the api from local React app?

File: controllers/admin/cohort.controller.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+const cohorts = require("../../dao/cohort.dao");
+
+function CohortController() {
+  this.index = async function store(req, res, next) {
+    const result = await cohorts.getCohorts();
+    res.render("./admin/index", { cohorts: result});
```
Comment (jpatrici) [inline controllers/admin/cohort.controller.js:6]: wondering if this will be replaced by the react route?

Comment (jpatrici) [inline controllers/admin/cohort.controller.js:6]: wondering if this will be replaced by the react route?

**Review by jpatrici — APPROVED** [review]

File: middlewares/auth.middleware.js (hunk 1 above)
Comment (theodorusandi) [reply middlewares/auth.middleware.js]: Thank you for the feedback

File: server.js (hunk 2 above)
Comment (theodorusandi) [reply server.js]: On production environement the backend only accepts request from Netlify. Now I set it to development.

**Pull request merged**